pkgs/racket-doc/scribblings/reference/pairs.scrbl

```diff
@@ -1255,6 +1255,28 @@ Returns a list with all elements from @racket[lst], randomly shuffled.
   (shuffle '(1 2 3 4 5 6))]}
 
 
+@defproc[(combinations [lst list?] [size (or/c #f exact-nonnegative-integer?) #f])
+         list?]{
+
+Return a list of all combinations of the input list.
```

Comment:
Maybe a like to WP or a quick explanation of "combination"? It's a heavily overloaded term, so it may not be clear with meaning you're using. I guess if you put these docs after `permutations`, that may be clear from context?